p4_symbolic/bmv2/bmv2.proto

```diff
@@ -49,7 +49,7 @@ enum ParserTransitionTypeEnum {
 
 // The type of a table.
 // Determines where actions are defined and how they get selected.
-enum TableTypeEnum {
+enum ActionSelectorType {
   simple = 0;       // A Simple key and action table.
```

Comment:
Use all-uppercase for this, see https://developers.google.com/protocol-buffers/docs/style#enums

Reply:
Same for all the other enums (sorry for not noticing earlier). And don't end enum names in *Enum.

Reply:
I can change the enum name, but I cannot change the content names of enums inside bmv2.proto. They must much (case sensitive) exactly the values in the JSON file as strings. Otherwise protobuf will be unable to parse the JSON for us.

The other enums (the ones in p4_symbolic/ir) that do not interact with the JSON directly have no such limitation, and I think their names/contents are fine now, but I will double check.

Reply:
Got it.

This can be a follow-up (or just an issue to remind us), but it may be nice for the IR to not reference things from phase 1. If something is shared, it should probably live in the IR, and be used in phase 1. And everything that is in IR should be following styleguides; if that means we need to have two enums (one for each phase), that's probably the right thing to do, rather than compromise on style for phase 2.

Reply:
This makes a lot of sense. I added it as an issue. Currently we only have one such enum re-used by IR. I will get rid of it soon.